testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/users/UserAttributesTest.java

```diff
@@ -99,4 +112,31 @@ public void disabledUser() {
         // TODO try to log in
     }
 
+    @Test
+    public void createUserWithGroups() {
```

Comment:
This test seem to be failing. Needs to be passing with FF and Chrome.

Reply:
fixed in latest commit